src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work");
```

Comment:
The message in the 'doWork' method should be more informative and indicate the type of machine that is working, e.g., 'Bulldozer is working'.

Reply:
Fixed